client/src/components/Login.js

```diff
@@ -19,14 +19,20 @@ const Login = props => {
     event.preventDefault();
     loginUser(email, password)
       .then(data => {
-        const token = data.token;
+        if (Object.keys(data).length === 0) {
+          setErrorMessage(
+            "The login details are not correct. Please try again"
+          );
+        } else {
+          const token = data.token;
```

Comment:
I'd remove the variable `token` as it's only used once and it doesn't make the code any clearer.